.github/workflows/main.yml

```diff
@@ -183,6 +183,18 @@ jobs:
     - run: cargo check --target wasm32-unknown-emscripten -p wasi-common
     - run: cargo check --target armv7-unknown-linux-gnueabihf -p wasi-common
 
+  # Check whether `wasmtime` cross-compiles to aarch64-pc-windows-msvc
```

Comment:
Can we add a note here, with a link to the open issue, noting that Wasmtime is known to have issues on this platform (trap handling) and this is why it doesn't have a build with artifacts?

Reply:
OK. I've opened a new issue to track the trap handling on Windows ARM64, and added the link.

Reply:
Thanks!